src/models/integrations.ts

```
@@ -0,0 +1,687 @@
+   * Apply data transformation
+   */
+  applyTransform: (value: any, transform: DataTransform): any => {
+    switch (transform.type) {
+      case 'uppercase':
+        return String(value).toUpperCase();
+      case 'lowercase':
+        return String(value).toLowerCase();
+      case 'trim':
+        return String(value).trim();
+      case 'substring':
+        const { start, end } = transform.config;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)